pages/cookbook/dexes/stonfi.mdx

```diff
@@ -1,13 +1,331 @@
 # STON.fi
 
-import { Callout, Steps, Tabs } from 'nextra/components'
-
-{/* See: https://nextra.site/docs/guide/built-ins */}
+import { Callout } from 'nextra/components'
 
 [STON.fi](https://ston.fi) is a decentralized automated market maker (AMM) built on [TON blockchain](https://ton.org) providing virtually zero fees, low slippage, an extremely easy interface, and direct integration with TON wallets.
 
-<Callout emoji="🚨">
+<Callout type="warning">
+  This api is about STON.fi version 2, which is under development. Use it at your own risk.
+</Callout>
+
+## Swaps
 
-  This page is a stub until it gets new content in [#149](https://github.com/tact-lang/tact-docs/issues/149).
+More about swaps [here](https://docs.ston.fi/docs/developer-section/api-reference-v2/example_swap).
 
-</Callout>
+Some variables like `offerAmount` are hardcoded for demonstration purposes. Don't forget to change them in real case scenarios.
```

Comment:
we should explain what `offerAmount` is and how it should be `changed in real case scenarios`. none of the below examples explain that. you can probably just add a link to the related stonfi doca page and tell readers to look there for other variables explanations too